src/examples/java/com/amazonaws/crypto/examples/EscrowedEncryptExample.java

```diff
@@ -36,24 +36,29 @@
  * <p>
  * Arguments:
  * <ol>
- * <li>KMS KeyArn
+ * <li>Key ARN: To find the Amazon Resource Name of your KMS customer master key (CMK), 
+ *     see 'Viewing Keys' at http://docs.aws.amazon.com/kms/latest/developerguide/viewing-keys.html
  * <li>File Name
  * </ol>
  *
- * Some organizations want the ability to decrypt their data even if KMS is unavailable. This
- * program demonstrates one possible way of accomplishing this by generating an "Escrow" RSA
- * key-pair and using that in addition to the KMS key for encryption. The organization would keep
- * the RSA private key someplace secure (such as an offline HSM) and distribute the public key their
- * developers. This way all standard use would use KMS for decryption, however the organization
- * maintains the ability to decrypt all ciphertexts in a completely offline manner.
+ * AWS Key Management Service (KMS) is highly available. However, some organizations want to decrypt
+ * their data offline and independent of KMS. This sample demonstrates one way to do this.
+ * 
+ * This program generates an "escrowed" RSA key pair. It stores the private key in a secure offline 
```

Comment:
The example code here does not store the private key durably anywhere; it is just held in memory for the life of the `EscrowedEncryptExample` object instance.  The intention of the original wording here was to indicate that if you were using the `JceMasterKey` class, your keys should probably be persisted somewhere so that you can actually use them outside of the context of a single object instance, and that the mechanism for storing those keys should be secure.